tests/present_and_fetch.rs

```diff
@@ -1,29 +1,20 @@
-// TODO: Once winit is updated again, restore this test.
-/*
 use softbuffer::{Context, Surface};
 use std::num::NonZeroU32;
 use winit::event_loop::EventLoopWindowTarget;
 
 fn all_red(elwt: &EventLoopWindowTarget<()>) {
-    let window = winit::window::WindowBuilder::new()
-        .with_title("all_red")
-        .build(elwt)
-        .unwrap();
+    #[cfg_attr(not(target_arch = "wasm32"), allow(unused_mut))]
+    let mut builder = winit::window::WindowBuilder::new().with_title("all_red");
 
     #[cfg(target_arch = "wasm32")]
     {
-        use winit::platform::web::WindowExtWebSys;
-
-        web_sys::window()
-            .unwrap()
-            .document()
-            .unwrap()
-            .body()
-            .unwrap()
-            .append_child(&window.canvas())
-            .unwrap();
+        use winit::platform::web::WindowBuilderExtWebSys;
+
+        builder = builder.with_append(true);
     }
 
+    let window = builder.build(elwt).unwrap();
+
     // winit does not wait for the window to be mapped... sigh
     #[cfg(not(target_arch = "wasm32"))]
     std::thread::sleep(std::time::Duration::from_millis(1));
```

Comment:
Just leaving this here as another argument for https://github.com/notgull/winit-test/issues/11: my system needed longer to work correctly.